presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/common/hadoop/hive_timestamp_nanos.properties

```diff
@@ -0,0 +1,16 @@
+connector.name=hive-hadoop2
+hive.metastore.uri=thrift://hadoop-master:9083
+hive.config.resources=/docker/presto-product-tests/conf/presto/etc/hive-default-fs-site.xml
+hive.allow-add-column=true
+hive.allow-drop-column=true
+hive.allow-rename-column=true
+hive.allow-comment-table=true
+hive.allow-comment-column=true
+hive.allow-drop-table=true
+hive.allow-rename-table=true
+hive.allow-register-partition-procedure=true
+hive.metastore-cache-ttl=0s
+hive.fs.cache.max-size=10
+hive.max-partitions-per-scan=100
```

Comment:
I guess. I took the `defautl` file and added last line.

Reply:
i figured.

the default hive.properties is not all-defaults, because we made it accommodate  all the many different tests we want to run on it
your new catalog can have -- and IMO should have -- as little configuration as possible

Reply:
I changed that. The catalog is now defined in test class.

Reply:
> I changed that. The catalog is now defined in test class.

it is still here, right?

(that's OK, i just don't understand)

Reply:
It was meant to be reply to https://github.com/prestosql/presto/pull/6424#discussion_r547335231. Sorry.

Reply:
As for this one, I will get rid of as many properties as I can.

Reply:
```
connector.name=hive-hadoop2
hive.metastore.uri=thrift://hadoop-master:9083
hive.config.resources=/docker/presto-product-tests/conf/presto/etc/hive-default-fs-site.xml
hive.allow-drop-table=true
hive.metastore-cache-ttl=0s
hive.translate-hive-views=true
hive.timestamp-precision=NANOSECONDS
```
seems a safe subset